### DISCUSSION TITLE
Matataki 作为内容平台的定位是什么？

### DISCUSSION THREAD

**Discussion by Linklinkguan** [Matataki-io/matataki-frontend#988] [Discussion]
瞬Matataki 是作为Fan票的试验园而诞生的，自诞生之初的目的就是为了Fan票的应用场景探索而服务，因此没有对其内容定位做过明确的定义。一直以来Matataki上的内容都是通过用户对平台调性的自发理解而主动选择的，我们作为开发团队没有做过任何引导。但是非常可喜的是，运营的2年多时间以来，极少会出现危险的内容。并且很多用户会选择在Matataki上撰写有深度思考的文章，主要有偏向于对科技、人文、货币、生活方式等有趣的领域。这可能也是由于大多数用户是被创始人 岛娘 吸引过来，或者是被 “内容永存”概念击中痛点，或是被FAN票（SocialMoney）概念所吸引过来，在当前这个时间节点下，无论是被哪点吸引过来的创作者都不会是纯粹的普通人，大多数都有着非常不一样的世界观、价值观，但是保有科技进步主义的态度，对未知的前沿技术充满了热忱。
我作为瞬Matataki的创始人之一，非常希望将创作者带给我们的内容特色就作为 瞬 的 平台内容定位。这些美好的感觉是从平台里经过两年时间生长出来的，不是我们用产品或运营的手段强加出来的。

## 为什么现在要提出对 内容定位 的讨论呢？
瞬Matataki 上的内容越来越多，增长的速度也越来越快，目前的“排大队”模式显然是无法适应创作者们对于“圈地自萌”的需求了。参考成功的UGC内容为主导的平台比如Bilibili、知乎，都对自己平台上的内容进行了大的板块划分，从而让创作者们可以在各自的板块中发挥实力，获得更多的影响力。板块可以理解为 层级更高的标签，可以用来塑造一个内容平台对内容的主观取向，因为发布的内容是必须要归类到某一种板块里的。

### 标签
Matataki中一直都有标签系统，但是缺乏对于 标签 的运营管理，同时在UX上也没有去强调利用 标签 来对内容进行划分的意义。因此即便是有着标签功能，也是没能实现让用户通过标签来发现更多 感兴趣 的 好内容 的效果。

## Matters
Matters是一个在内容运营上比较成功的产品，他们在内容架构上主打的就是 “讨论”“文章引用”“随处@”的能力，从而将每一篇文章的评论区都打造成了一个可以进行深度交流的场所。以此构建出了一个非常强互动的社区形态。

**Comment by whyouare111** [comment]
几种模式会涉及到公共舆论场和私人言论空间的问题。赛博空间如果在应用层都是免费使用别人的资源搭起来的，技术层面上就不适合作为保管数字资产的空间，最多就是个传播渠道。
从历史经验看，即使是Google Reader，Google Plus这类服务，虽然用起来像公共产品，清算的时候只会被算作是Google处置自己的私有财产吧，Google想关在法理上也是没有任何问题，我们在免费使用Google的资源罢了。至于服务吸纳的数据给不给时间迁移，恐怕只能看服务提供商开不开恩而已
借助那些定位是永在网的技术（如IPFS)一定程度上是能解决这个问题，不过永在网的网关+节点们没有义务替具体某个人缓存/保存TA的数字资产，结果还是他力本愿模式。配合Filecoin类变成支付保管费的模式的话，在马太效应下最后会不会只是大矿机主抢了CDN供应商的生态位，一看实际的存储介质还是高度中心化的，只是依附关系变了而已呢？
